ExecuteJobs.java

```diff
@@ -0,0 +1,80 @@
+import java.util.*;
+import java.io.*;
+
+public class ExecuteJobs {
```

Comment:
for the final submission we'd be asking for more docs

Reply:
I'm planning on adding docs once the code is more finalized.